packages/query/src/queries/get-route-metadata.graphql

```diff
@@ -0,0 +1,36 @@
+query GetRouteMetadata($uri: String!) {
+	nodeByUri(uri: $uri) {
```

Comment:
1. `templateByUri`
2. These should be built up holistically with fragments + interfaces (`NodeWithTitle`, `NodeWithAuthor` etc)
3. use global `id`s on every node.